core/src/components/datetime/datetime.md.scss

```diff
@@ -36,7 +36,7 @@
 
 :host .calendar-action-buttons ion-item,
 :host .calendar-action-buttons ion-button {
-  color: #{$text-color-step-350};
+  --color: #{$text-color-step-350};
```

Comment:
Since we are exposing the `ion-item` instance we should use the public API for setting the color otherwise developers will run into issues changing the color using `--color`.

Reply:
Turns out this also fixes a bug where the expanded state of the month/year toggle never changed color! This is why there are screenshot diffs here.

Example: https://ionicframework.com/docs/api/datetime#basic-usage

Switch to MD mode and click the month/year button. Observe that the color of the button only changes to blue when you hover over it. Repeat the same steps on iOS mode. Observe that the color of the button correctly changes to blue regardless of hover state when expanded.